src/app/app-routing.module.ts

```diff
@@ -1,66 +1,56 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AppRoutes } from '@Enums/routes.enum';
-import { NotFoundComponent } from './shared/pages/not-found/not-found.component';
-import { SettingGoalsComponent } from './setting-goals/setting-goals.component';
-import { EducationCertificationsComponent } from './education-certifications/education-certifications.component';
-import { ReimbursementOfExpensesComponent } from './reimbursement-of-expenses/reimbursement-of-expenses.component';
-import { FeedbackFormComponent } from './feedback-form/feedback-form.component';
-import { OnboardingPlanComponent } from './onboarding-plan/onboarding-plan.component';
-import { PaidTimeOffComponent } from './paid-time-off/paid-time-off.component';
-import { PerformanceEvaluationComponent } from './performance-evaluation/performance-evaluation.component';
-import { SelfServicePortalComponent } from './self-service-portal/self-service-portal.component';
-import { HomeComponent } from './home/home.component';
 
 const routes: Routes = [
   {
     path: AppRoutes.Empty,
-    redirectTo: AppRoutes.HomePage,
+    loadComponent: () => import('./home/home.component').then((x) => x.HomeComponent),
```

Comment:
you can keep the redirect as is. It will automatically go to the home route and lazy load the home component from there :D

Reply:
Fixed!